newsroom/push.py

```diff
@@ -767,6 +767,10 @@ def notify_new_agenda_item(_id, check_topics=True):
     with locked(_id, "agenda"):
         agenda = app.data.find_one("agenda", req=None, _id=_id)
         if agenda:
+            if agenda.get("recurrence_id") and agenda.get("recurrence_id") != _id:
```

Comment:
How will this work if one or more Event(s) are updated in the series that does't include the original?

**Example scenario:**
There is a sporting match that goes for 5 days.
The game is finished on the 3rd day.
The planner cancels the 4th and 5th occurrence of the Event in Superdesk, which subsequently cancels the associated Planning & Coverages for those days.

**Current Functionality**
2 emails would be sent to subscribers, one for each Event that was cancelled

**New Functionality (based on this PR)**
No emails would be sent to subscribers

Reply:
yep you're right, will ignore the other events only when agenda item is created for the first time